terraform/aws/components/gp2gp/s3.tf

```diff
@@ -0,0 +1,59 @@
+
+# S3 bucket for storing extracts as cache
+resource "aws_s3_bucket" "gp2gp_extract_cache_bucket" {
+  bucket = "${var.environment_id}-gp2gp-extract-cache-bucket"
+  tags = {
+    Name = "${var.environment_id}-gp2gp-extract-cache-bucket"
```

Comment:
Same as for resource name, tags for resources are also combined in locals. The value for tags should be: 
```
  tags = merge(local.default_tags, {
    Name = "${local.resource_prefix}-extract_cache_bucket"
  })
```

Reply:
thanks for reminding, I missed tags. what about EnvironmentId tag?

Reply:
The environment tag is added in locals, see:
```
locals {
  default_tags = {
    Project = var.project
    Environment = var.environment
    Component = var.component
  }
```

Reply:
local tags merged.